src/lib/profiles/data-management/Current/NotificationEngine.cpp

```diff
@@ -1701,6 +1701,18 @@ WEAVE_ERROR NotificationEngine::BuildSubscriptionlessNotification(PacketBuffer *
 
 #endif // WDM_ENABLE_SUBSCRIPTIONLESS_NOTIFICATION
 
+void NotificationEngine::OnNotifyScheduleWorkCallback(System::Layer * aSystemLayer, void * aAppState, System::Error)
+{
+    NotificationEngine * const pEngine = reinterpret_cast<NotificationEngine *>(aAppState);
+    WeaveLogDetail(DataManagement, "%s", __func__);
```

Comment:
No need for additional logging here.  It doesn't add anything valuable.

Reply:
Done, thanks